static/app/views/replays/replayTable/index.tsx

```diff
@@ -207,12 +244,15 @@ const StyledPanelTable = styled(PanelTable)<{
       )
       .join(' ')};
 
-  ${props => (props.gridRows ? `grid-template-rows: ${props.gridRows};` : '')}
+  ${props =>
+    props.gridRows
+      ? `grid-template-rows: ${props.gridRows};`
+      : 'grid-template-rows: 44px max-content'}
 `;
 
 const StyledAlert = styled(Alert)`
-  border-radius: 0;
-  border-width: 1px 0 0 0;
+  border-radius: 0px;
+  border-width: 1px 0 1px 0;
```

Comment:
the styled alert looks better inside the cards 👍 

can you also check how it looks inside the main table? that might not have a min-height but idk how the extra border will be.

Reply:
Looks like our issues stream does not include the top border: 
![image](https://github.com/getsentry/sentry/assets/79684/c09acd6c-2e21-4517-ae37-80a5d0c7b075)

Reply:
do we want to remove the top border for both alerts here?

Reply:
<img width="1165" alt="SCR-20230727-kwsp" src="https://github.com/getsentry/sentry/assets/56095982/aee5e6dd-2e92-4a66-8502-754946219c53">
rn looks like this

Reply:
issue stream also has that margin. i don't love the margin though

Reply:
i can add in that margin to the style for continuity

Reply:
<img width="1116" alt="SCR-20230727-lzwi" src="https://github.com/getsentry/sentry/assets/56095982/bbbdd803-944b-48ab-a68c-6607863927ee">
how's this? i tried removing top border too but it looks weird without the top border

Reply:
discussed offline: @ryan953 suggested keeping rage click table alert as is, but removing margin & bottom border for big replay table